app/README.md

```diff
@@ -100,3 +102,35 @@ Please be sure to read the style guides and best practices within the above docu
 React Navigation is used to provide "native"-feeling navigation for the application.
 
 See the React Navigation documentation [here](https://reactnavigation.org/docs/getting-started/).
+
+### Deploying Android and iOS to AppCenter
+
+The Twilio Voice React Native App uses github tags to deploy both Android and iOS apps to AppCenter. The github tags contain options for either a "rc" or "release". "rc" and "release" options can also contain "preview" and "beta" filters. Append "-android" or "-ios" to the end of tags for platform specific release.
+
+#### iOS and Android example release
```

Comment:
From the config.yml it seems there is no common tag filter that starts pipeline for both platforms?

Reply:
ah I missed that. I noticed this release you did: https://github.com/twilio/twilio-voice-react-native-app/releases/tag/0.1.1-beta.1-rc2 . But this must have been from when you were creating the PR?

Reply:
yup this was later overridden by the Android release updates